.github/workflows/main.yml

```diff
@@ -8,11 +8,18 @@ jobs:
     runs-on: ${{matrix.os}}
     steps:
       - uses: actions/checkout@v2
-      - uses: dcodeIO/setup-node-nvm@master
+      - name: Use Node.js ${{matrix.node}}
+        uses: actions/setup-node@v2
         with:
           node-version: ${{matrix.node}}
+          #cache: npm
+      - run: npm install -g npm@7 # https://github.com/npm/cli/issues/4341#issuecomment-1040608101
       - run: npm install -g npm
       - run: npm install
+      - name: Cherry pick https://github.com/wooorm/trough/pull/11
+        run: |
+          curl https://patch-diff.githubusercontent.com/raw/wooorm/trough/pull/11.patch |
+          git apply --ignore-space-change --directory node_modules/trough
```

Comment:
I think passing the `this` value (the processor) from unified-lint-rule -> the rule means `wrap()` needs to preserve `this` as well? I opened https://github.com/wooorm/trough/pull/11 for that. In the CI, I apply that patch -> `node_modules` so that the checks run to completion.

Reply:
I dropped this commit, now that https://github.com/wooorm/trough/pull/11 has been released. :heart: